inc/admin/ui/modules/optimization.php

```diff
@@ -202,7 +216,7 @@
 			'type'         => 'textarea',
 			'label_for'    => 'exclude_css',
 			'label_screen' => __( 'CSS files to be excluded from minification and concatenation:', 'rocket' ),
-			'placeholder'  => "e.g. /wp-content/themes/some-theme/style.css\ne.g. /wp-content/plugins/some-plugin/(.*).css",
+			'placeholder'  => "/wp-content/themes/some-theme/style.css\n/wp-content/plugins/some-plugin/(.*).css",
```

Comment:
Done in https://github.com/wp-media/wp-rocket/pull/418/commits/6696b048b0799dbd6882b820ee493ae18814fa38